### PR TITLE
Fixed Wunderbar Professions, VehicleBar

### LINKED ISSUE #8: Wunderbar invisible at start
### Description

Wunderbar is invisible at login or reload need to change anything to make it visible again like texture, bg color etc. Tested only with ElvUi, Windtools and ToxiUi

### Screenshots

image

### To reproduce

1.

### Acceptance criteria

-   [ ]

### PR DESCRIPTION
**Pull request by Wolen**
Fixed #8 and fixed Vehiclebars not working.

### CODE REVIEW

File: Modules/WunderBar/SubModules/DataBar.lua [hunk 1]
```diff
@@ -13,7 +13,7 @@ local C_Reputation_GetFactionParagonInfo = C_Reputation.GetFactionParagonInfo
 local C_Reputation_IsFactionParagon = C_Reputation.IsFactionParagon
 local CreateFrame = CreateFrame
 local format = string.format
-local GetFriendshipReputation = GetFriendshipReputation
+local GetFriendshipReputation = C_GossipInfo.GetFriendshipReputation
```
Comment (bagelbits) [inline Modules/WunderBar/SubModules/DataBar.lua:16]: I'm betting that `C_GossipInfo` needs to be added to the `.luacheckrc` file as a `global`. If you look for `C_QuestLog` in that file, it'll probably be within that same block.

Comment (bagelbits) [reply Modules/WunderBar/SubModules/DataBar.lua:16]: Sweet!

**Comment by Toxicom** [comment]
Need to solve the razorgore pet/vehicle bullshit and good to go?

**Comment by bagelbits** [comment]
What's going on with that one?

**Comment by Toxicom** [comment]
https://discord.com/channels/769550106948141086/1036023658075934720/1036692829910601739

File: Core/InternalData.lua [hunk 2]
```diff
@@ -56,8 +56,14 @@ I.Data.Contributor = {
 
     [I.Enum.Developers.WOLEN] = {
       [I.Enum.Flavor.RETAIL] = {
-        -- yes
-        ["Wolen-SomeRealm"] = true,
+        -- Mal'Ganis
+        ["Wolen-Mal'Ganis"] = true,
```
Comment (Toxicom) [inline Core/InternalData.lua:60]: Don't think `'` is valid, can you confirm these are the exact realm names with `/tx export names` ?

**Pull request merged**